pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -1207,11 +1207,17 @@ func createRewriteFilter(filter *k8s.HTTPURLRewriteFilter) *istio.HTTPRewrite {
 	if filter.Path != nil {
 		switch filter.Path.Type {
 		case k8sbeta.PrefixMatchHTTPPathModifier:
-			rewrite.Uri = *filter.Path.ReplacePrefixMatch
+			if filter.Path.ReplacePrefixMatch != nil {
+				rewrite.Uri = *filter.Path.ReplacePrefixMatch
```

Comment:
Does not gateway api admission webhook has already made sure it is not nil

Reply:
FYI: https://github.com/zirain-dev/gateway-api/blob/12dec58a39548474d4ed6fb2f2d5f8d59cf53be7/apis/v1beta1/validation/httproute.go#L289

Reply:
Yes, It has been sure in validateHTTPPathModifier